src/util/regenerator-runtime.js

```diff
@@ -0,0 +1,6 @@
+import regeneratorRuntime from 'regenerator-runtime'; // Puts regeneratorRuntime into global scope for async/await
```

Comment:
babel transforms `async` to `generator`, `regenerator-runtime` is required for `generator` transpiled to es5

Reply:
Instead of this file, I believe you can just directly do
`import 'regenerator-runtime/runtime';`

Reply:
done